src/components/HyloTipTapEditor/HyloTipTapEditorMenuBar.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react'
```

Comment:
This is the top menu bar for the editor, currently only used in Web `PostEditor`. It's pretty easy to add/remove items here as well as re-style it. It exists independent of the editor other than needing a `ref` to it to call commands.